mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -447,14 +346,17 @@ def wkf_cancel(self, cr, uid, ids, context=None):
     def wkf_close(self, cr, uid, ids, context=None):
         """Change state from in progress to closed"""
         o = self.browse(cr, uid, ids, context=context)[0]
-        done_states = ['done', 'cancelled']
+        # TODO make it more friendly
+        done_states = ['done', 'cancelled', 'settled', 'rejected']
```

Comment:
This is a problem, I wasn't sure how to handle this. The problem come from the fact that those states are on a `mgmtsystem.action` object. The `mgmtsystem.action` object is a `crm.claim object`. But the `crm.claim` object doesn't have states anymore but stages.